src/app/components/avatar/avatar.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="card col-sm-2" (click)="navigateHero(hero.id, hero.name)">
+    <img src="{{hero.smallImage}}" class="card-img-top" *ngIf="hero">
+    <div class="card-footer">
+        <p class="card-title" *ngIf="hero">{{hero.name}} </p>
+    <p class="card-title finalWords" *ngIf="hero">"{{hero.finalWords}}"</p>
```

Comment:
same ngIf. use it in the parent tag once

Reply:
fixed, cheers